docs/skipped-urls.txt

```diff
@@ -14,5 +14,13 @@ https://c.s-microsoft.com/static/fonts/segoe-ui/west-european/*
 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort*
 https://github.com/npm/node-semver*
 
+# Any URL for the Github repo with an fragment that points to a line number in code.
+# No markdowwn anchor for the line will exist, but they're valid.
+https://github.com/microsoft/FluidFramework/.*#L\d+$
```

Comment:
Thanks for adding this! Good to have, although we should probably discourage linking to line numbers.